### ISSUE TITLE
Push default export to global scope for typescript files

### ISSUE THREAD

**Issue by Dsupreme** [webpack-contrib/expose-loader#82]
* Operating System: OS X
* Node Version: 10.12.0
* NPM Version: 6.5.0
* webpack Version: "4.28.0",
* expose-loader Version: "0.7.5"

This issue is for a:

- [ ] **bug**
- [x] **feature** request
- [ ] **modification** request

##### CLI Command

```bash
   webpack --mode=development
```

##### webpack.config.js

```js
module: {
        rules: [{
                test: /\.ts$/,
                include: path.join(__dirname, "src"),
                loader: "ts-loader"
            },
            {
                type: "javascript/auto",
                test: /\.json$/,
                loader: 'json-loader'
            },
            {
                test: require.resolve('./src/abc/index.ts'),
                loader: 'expose-loader?FP'
            }
        ]
    },
```

```js
 export default class abc {
    // do something
}
```

### Expected Behavior

abc is added to the global scope or under **window**

### Actual Behavior

Module parse failed: Unexpected token (11:16)
You may need an appropriate loader to handle this file type.

`export default class abc {} @ ./src/abc/index.ts-exposed 1:32-55`

### New Feature Use Case

When trying to use a class inside webpack generated bundle inside another js file (not under the scope of webpack)

**Comment by patrick-mcdougle** [comment]
This might be fixed by #85 I think the problem is that the expose loader doesn't know how to deal with harmony (ES) modules.

**Comment by cap-Bernardito** [comment]
A same problem #56. Will be fixed there.
